client/marketing/components/knowledge-base/test/index.js

```diff
@@ -49,11 +49,11 @@ describe( 'Posts and not loading', () => {
 		);
 	} );
 
-	it( 'should not display the spinner', () => {
+	it( 'should not display the preloader', () => {
 		const { container } = knowledgeBaseWrapper;
 		expect(
-			container.getElementsByClassName( 'components-spinner' )
-		).toHaveLength( 0 );
+			container.querySelector( '.is-loading, .is-preloader' )
```

Comment:
@tomalec , I don't see any other files here using `is-preloader` class name, I suppose it comes from other components? (Just want to be sure this is not typo / confusion with `is-placeholder`.

Reply:
Thanks :) It was a typo.